pkg/ccl/backupccl/testdata/backup-restore/restore-protect-spans

```diff
@@ -0,0 +1,55 @@
+
+# tickle the unprotected timestamp bug
```

Comment:
this test is an attempt to tickle the original bug we observed in that big restore. in this commit, the PTS protection step in the restore job is commented out, so I'd expect the bug to get tickled. But maybe i'm not toggling the right cluster settings?

Reply:
Test has been removed